dbt/include/clickhouse/macros/materializations/table.sql

```diff
@@ -0,0 +1,85 @@
+{% materialization table, default %}
```

Comment:
Should this be `adapter='clickhouse'` instead of `default` just for consistency?